Cloud Application Manager/Getting Started/register-existing-google-cloud-instance.md

```diff
@@ -0,0 +1,85 @@
+{{{
+"title": "Register an Existing Google Cloud Instance",
+"date": "09-30-2019",
+"author": "Efren Rey",
+"keywords": ["cam", "instances", "register", "google"],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Find a Google Cloud instance](#find-a-google-cloud-instance)
+* [Register the instance](#register-the-instance)
+* [Install the Agent Installation Script manually](#install-the-agent-installation-script-manually)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+This tutorial will show you how to import an existing instance that was discovered in your *Google Cloud* provider using [Cloud Application Manager](https://www.ctl.io/cloud-application-manager).
+For further information about instance registering using *Cloud Application Manager*, please visit [Register an Existing Instance](./register-existing-instance.md).
+
+### Audience
+
+Users who want to register their *Google Cloud* instances into *Cloud Application Manager* to enable lifecycle management on them.
+
+### Prerequisites
+
+* A *Google Cloud* machine
+* An active *Cloud Application Manager* account 
+* An existing *Google Cloud* account with active instances
+
+### Find a Google Cloud instance
+
+Within the *Cloud Application Manager* workspace, navigate to your *Google Cloud* provider and run the sync function by clicking **Sync**.
+
+The sync process will discover unregistered *Google Cloud* instances and list them under the **Unregistered Instances** tab.
+
+![Locate Unregistered Instances](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-1.png)
+
+### Register the instance
+
+Under the **Unregistered Instances** tab or within the *Instances* page, click the instance you want to import and register.
+
+Provide any details for the instance in the fields provided.
+
+![Register Instance](../../images/cloud-application-manager/register/cam-register-existing-google-cloud-instance-2.png)
+
+The automatic registration process will require reboot the virtual machines, or regenerated in case of an scaling group.
+To avoid that restarts, the *Install Agent Manually* togle can be enabled to install manually the CAM agent.
```

Comment:
```suggestion
To avoid the restart operation, the *Install Agent Manually* toggle can be enabled to install manually the Cloud Application Manager (CAM) agent.
```